stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,562 @@
+                                    aria-haspopup="true" onclick="onPopoverClick('actionSheetDesktopRTL');">
+                                <i class="sap-icon--overflow"></i>
+                            </button>
+                        </div>
+                        <div class="fd-popover__body" aria-hidden="true" id="actionSheetDesktopRTL">
+                            <ul class="fd-action-sheet fd-action-sheet--compact" role="list" aria-label="List of contextual options">
+                                <li class="fd-action-sheet__item" role="listitem">
+                                    <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                                        <i class="sap-icon--attachment"></i>
+                                        <span class="fd-button__text">Button 1</span>
+                                    </button>
+                                </li>
+                                <li class="fd-action-sheet__item" role="listitem">
+                                    <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                                        <i class="sap-icon--add"></i>
+                                        <span class="fd-button__text">Button 2</span>
+                                    </button>
+                                </li>
+                                <li class="fd-action-sheet__item" role="listitem">
+                                    <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                                        <i class="sap-icon--print"></i>
+                                        <span class="fd-button__text">Button 3</span>
+                                    </button>
+                                </li>
+                            </ul>
+                            </div>
+                        </div>
+                    </div>
+            
+                <p dir="rtl" class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.</span>
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--byline">Aug 20, 2020</small>
+                    <small class="fd-feed-list__footer--byline">Reply</small>
+                </div>
+            </div>          
+        </li>    
+    </ul>
+</div>
+`;
+
+rtl.storyName = 'RTL';
+rtl.parameters = {
+    docs: {
+        storyDescription: 'The feed list item can display from right to left on the screen, so that it may be used internationally.'
+    }
+};

```

Comment:
We can skip the visual RTL snapshot for RTL story.

```suggestion
    docs: {
        storyDescription: 'The feed list item can display from right to left on the screen, so that it may be used internationally.'
    },
    skipRTLSnapshot: true
};

```

Reply:
Added